portalnet/src/storage.rs

```diff
@@ -1140,6 +1140,8 @@ pub mod test {
         // The restarted store should have the same radius as the original
         assert_eq!(radius, new_storage.radius);
 
+        drop(storage);
+        drop(new_storage);
```

Comment:
Why were the explicit drops necessary?

Reply:
They are in the tests below and above. This change isn't related to the PR. Without these drops the test will panic on certain operating systems. If you read the doc for .close() on the line before there is a comment in the example about this.